packages/frontend-shared/src/gql-components/RecordPrompt.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <p class="mb-24px">
```

Comment:
Leave the copy block out of the RecordPrompt component so that the surrounding copy can be customized where it is used.